### PR TITLE
refactor: use ox_lib, qbx lib, and remove PolyZone

### LINKED ISSUE #7: Get rid of polyzone
### The problem

polyzone

### Ideal solution

ox_lib please someone :)

### PR DESCRIPTION
**Pull request by solareon**
## Description

Replaces locales with ox_lib locales. Removes polyzone dependency in exchange for ox_lib zones and ox_target. Updates to qbx lib.

## Checklist

- [ ] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x] My pull request fits the contribution guidelines & code conventions.

### CODE REVIEW

**Comment by solareon** [comment]
fixes #7

File: server/main.lua [hunk 1]
```diff
@@ -76,25 +76,26 @@ RegisterNetEvent('qb-tow:server:11101110', function(drops)
...
+    help = locale("info.tow"),
 }, function(source)
     local Player = exports.qbx_core:GetPlayer(source)
     if Player.PlayerData.job.name ~= "tow" and Player.PlayerData.job.name ~= "mechanic" then return end
     TriggerClientEvent("qb-tow:client:TowVehicle", source)
 end)
 
 lib.callback.register('qb-tow:server:spawnVehicle', function(source, model, coords, warp)
-    local netId = SpawnVehicle(source, model, coords, warp)
+    local warpPed = warp and GetPlayerPed(source) or false
```
Comment (BerkieBb) [inline server/main.lua]: The `or false` is not needed, does it automatically already

File: client/main.lua [hunk 2]
```diff
@@ -81,86 +81,69 @@ local function CreateZone(type, number)
 
     if type == "main" then
         event = "qb-tow:client:PaySlip"
-        label = Lang:t("label.payslip")
+        label = locale("label.payslip")
         coords = sharedConfig.locations["main"].coords.xyz
         heading = sharedConfig.locations["main"].coords.w
         boxName = sharedConfig.locations["main"].label
-        size = 3
+        size = vec3(3,3,10)
```
Comment (BerkieBb) [inline client/main.lua]: spaces are allowed

File: client/main.lua [hunk 3]
```diff
@@ -81,86 +81,69 @@ local function CreateZone(type, number)
...
-            end
-        end)
+        local zone = lib.zones.box({
+            coords = coords,
+            size = size,
+            rotation = heading,
+            debug = config.debugPoly,
+            event = event,
+            inside = function(self)
+                TriggerEvent(self.event)
```
Comment (BerkieBb) [inline client/main.lua]: can just use event directly

File: client/main.lua [hunk 4]
```diff
@@ -81,86 +81,69 @@ local function CreateZone(type, number)
...
+            local zoneMark = lib.zones.box({
+                coords = coords,
+                size = vec3(20,20,10),
+                rotation = heading,
+                debug = config.debugPoly,
+                onEnter = function(self)
                     TriggerEvent('qb-tow:client:ShowMarker', true)
-                else
+                end,
+                onExit = function(self)
```
Comment (BerkieBb) [inline client/main.lua]: unused self

File: client/main.lua [hunk 5]
```diff
@@ -81,86 +81,69 @@ local function CreateZone(type, number)
...
-                end
-            end
-        end)
+        local zone = lib.zones.box({
+            coords = coords,
+            size = size,
+            rotation = heading,
+            debug = config.debugPoly,
+            event = event,
+            inside = function(self)
```
Comment (jnccloud) [inline client/main.lua]: I had to change this to `onEnter` or else the context menu spams

**Comment by jnccloud** [comment]
I had to add `ox_lib 'locale'` to `fxmanifest.lua` or else it throws a locale exception.

File: client/main.lua (hunk 5 above)
Comment (solareon) [reply client/main.lua]: good catch

File: server/main.lua [hunk 6]
```diff
@@ -11,12 +11,12 @@ RegisterNetEvent('qb-tow:server:DoBail', function(bool, vehInfo)
         if not Bail[Player.PlayerData.citizenid] then return end
         Player.Functions.AddMoney('bank', Bail[Player.PlayerData.citizenid], "tow-bail-paid")
         Bail[Player.PlayerData.citizenid] = nil
-        TriggerClientEvent('ox_lib:notify', src, {description = Lang:t("success.refund_to_cash", { value = config.bailPrice }), type = 'success'})
+        TriggerClientEvent('ox_lib:notify', src, {description = locale("success.refund_to_cash", config.bailPrice ), type = 'success'})
```
Comment (TonybynMp4) [inline server/main.lua:14]: core export plz :c

**Review by TonybynMp4 — APPROVED** [review]
Apart from the gameplay problems this PR works fine 👍🏻

**Review by Randolio — APPROVED** [review]

**Pull request merged**